tests/cases/compiler/commentsdoNotEmitComments.ts

```diff
@@ -88,3 +88,8 @@ module m1 {
 
 /// this is x
 declare var x;
+
+
+/** const enum member value comment (generated by TS) */

```

Comment:
The test is that the removeComments flag causes comments not to be emitted. So the point here is that there should be no comments in the javascript emitted for this test code. Could you explain why this is a seperate test case.